dotCMS/src/main/java/com/dotcms/rest/api/v2/languages/LanguagesResource.java

```diff
@@ -410,6 +408,30 @@ public Response getAllMessages (
         return Response.ok(new ResponseEntityView(result)).build();
     }
 
+    @GET
+    @Path("/variables")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public Response getVariables(
+            @Context final HttpServletRequest request,
+            @Context final HttpServletResponse response,
+            @BeanParam final PaginationContext paginationContext) throws DotDataException {
```

Comment:
keep in mind we already have some standard to do pagination by PaginationUtil, take a look